.gitignore

```
@@ -0,0 +1,2 @@
+# This is the most effective .gitignore ever! It ignores... nocode.
+!*
```

this leaves your repo vulnerable to contributors committing new code
```suggestion
*
```